dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -102,7 +102,7 @@ const ShowSignInGate = ({
 	useEffect(() => {
 		submitViewEventTracking({
 			component: withComponentId(
-				signInGateTestIdToComponentId[abTest.id],
+				signInGateTestIdToComponentId[abTest.id] ?? 'unknown',
```

Comment:
@guardian/identity you may want to have a peek.

Reply:
just curious about the choice of `unknown` as a default here — is this a standard thing?

Reply:
Not particularly, hence the nudge.

I'm not sure what would make the most sense here, maybe you want to throw an error?